From a500fb1c1bad9865bd1bb5cd084924062feb896e Mon Sep 17 00:00:00 2001
From: Herbert Xu <herbert@gondor.apana.org.au>
Date: Tue, 20 Nov 2018 10:09:26 +0800
Subject: [PATCH 40/53] eval: Use sh_warnx instead of warnx

This patch fixes a typo in evalbltin where warnx was used instead
of sh_warnx.

Reported-by: Antonio Ospite <ao2@ao2.it>
Fixes: 8e43729547b5 ("eval: Report I/O error on stdout")

Signed-off-by: Herbert Xu <herbert@gondor.apana.org.au>
---
 src/eval.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/eval.c b/src/eval.c
index 8f65d94..f45e2e2 100644
--- a/src/eval.c
+++ b/src/eval.c
@@ -953,7 +953,7 @@ evalbltin(const struct builtincmd *cmd, int argc, char **argv, int flags)
 		status = (*cmd->builtin)(argc, argv);
 	flushall();
 	if (outerr(out1))
-		warnx("%s: I/O error", commandname);
+		sh_warnx("%s: I/O error", commandname);
 	status |= outerr(out1);
 	exitstatus = status;
 cmddone:
-- 
2.25.0

